### PR TITLE
MCR-2130 remove xerces, xml-resolver and xom dependency from mycore

### PR DESCRIPTION
**Pull request by yagee-de**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2130)

Success if `mvn dependency:tree -pl mycore-meta` do not return xerces, xml-resolver or xom anymore. Keep open until fully resolved.

### CODE REVIEW

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4630175&bid=15521007)

File: mycore-base/src/main/java/org/mycore/common/xml/MCREntityResolver.java [hunk 1]
```diff
@@ -274,4 +279,13 @@ public InputSource newInputSource() {
             return is;
         }
     }
+
+    private static class CatalogEntityIdentifier {
+        private String systemId, publicId;
```
Comment (adlerre) [inline mycore-base/src/main/java/org/mycore/common/xml/MCREntityResolver.java]: Codacy Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4630175)

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 9722b3b9cadf0f0008303f45cdf26cd45f24d4a0! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5145-pull_request-895

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 8fe65e3f3bedc2391c1de18dc7fa351f140c2ce5! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5166-pull_request-895

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on aa702bfe8adb740d217659dd5a5127e5b0e5ccfc! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5168-pull_request-895

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on aa702bfe8adb740d217659dd5a5127e5b0e5ccfc! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5168-pull_request-895

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 504580b5d3b74247c1451ee81e878667d7e09274! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5170-pull_request-895

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 504580b5d3b74247c1451ee81e878667d7e09274! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5170-pull_request-895

File: mycore-mods/src/main/java/org/mycore/mods/classification/MCRClassMapper.java [hunk 2]
```diff
@@ -215,8 +159,8 @@ private static MCRAuthorityInfo getAuthInfo(MCRCategoryID categID, String elemen
                 authInfo = new MCRTypeOfResource(categID.getID().replace('_', ' '));
             }
             if (authInfo == null) {
-                boolean supportCode = AUTHORITY_ELEMENTS.contains(elementLocalName);
-                boolean supportURI = AUTHORITY_URI_ELEMENTS.contains(elementLocalName);
+                boolean supportCode = SUPPORTED.contains(elementLocalName);
+                boolean supportURI = SUPPORTED.contains(elementLocalName);
```
Comment (rsteph-de) [inline mycore-mods/src/main/java/org/mycore/mods/classification/MCRClassMapper.java]: now 2 variables with the same value -> can the code be simplified?
boolean supported = SUPPORTED.contains( ... )

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 9bf10cffab4957de4d7c1d6f5531554dce837a30! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5191-pull_request-895

**Comment by MyCoRe-Travis** [comment]
Something gone wrong on 9bf10cffab4957de4d7c1d6f5531554dce837a30! 
 Please Check https://github.com/MyCoRe-Travis/test_artifacts/tree/5191-pull_request-895

File: pom.xml [hunk 3]
```diff
@@ -1446,7 +1446,7 @@
       <dependency>
         <groupId>org.mycore.mets</groupId>
         <artifactId>mets-model</artifactId>
-        <version>0.13</version>
+        <version>2.0-SNAPSHOT</version>
```
Comment (rsteph-de) [inline pom.xml:1449]: I would use a released version instead of "*-SNAPSHOT".

File: pom.xml [hunk 4]
```diff
@@ -1456,7 +1456,7 @@
       <dependency>
         <groupId>org.mycore.sword</groupId>
         <artifactId>easy-sword2-lib</artifactId>
-        <version>1.0.4</version>
+        <version>1.0.5-SNAPSHOT</version>
```
Comment (rsteph-de) [inline pom.xml:1459]: I would use a released version instead of "*-SNAPSHOT".

Comment (yagee-de) [reply pom.xml:1459]: will be released in 2020-02

**Review by rsteph-de — APPROVED** [review]
looks fine for me

**Pull request merged**